server/api/crud/artifacts.py

```diff
@@ -275,6 +275,12 @@ def _resolve_artifact_size(artifact, auth_info):
         if "spec" in artifact and "inline" in artifact["spec"]:
             validate_inline_artifact_body_size(artifact["spec"]["inline"])
 
+    @staticmethod
+    def _is_single_file(target_path):
+        if target_path:
+            return target_path.endswith(".parquet") or target_path.endswith(".pq")
```

Comment:
The function name doesn't match to what the function does.
I mean it didn't check if there is single file but also check the extension of this file.